contracts/test/RFQ.t.sol

```diff
@@ -508,11 +603,76 @@ contract RFQTest is StrategySharedSetup {
         sig = abi.encodePacked(r, s, v, bytes32(0), uint8(sigType));
     }
 
+    function _createSpenderPermitFromOrder(RFQLibEIP712.Order memory defaultOrder)
+        internal
+        view
+        returns (SpenderLibEIP712.SpendWithPermit memory makerAssetPermit, SpenderLibEIP712.SpendWithPermit memory takerAssetPermit)
+    {
+        // maker (= mm) order: -> taker recive maker's token but except fee
+        makerAssetPermit = SpenderLibEIP712.SpendWithPermit(
+            defaultOrder.makerAssetAddr,
+            address(rfq),
+            defaultOrder.makerAddr,
+            address(rfq),
+            defaultOrder.makerAssetAmount,
+            RFQLibEIP712._getOrderHash(defaultOrder),
+            uint64(defaultOrder.deadline)
+        );
+
+        // taker (= user) transaction (= fill): -> maker recive taker's token totally
+        takerAssetPermit = SpenderLibEIP712.SpendWithPermit(
+            defaultOrder.takerAssetAddr,
+            address(rfq),
+            defaultOrder.takerAddr,
+            address(rfq),
+            defaultOrder.takerAssetAmount,
+            RFQLibEIP712._getTransactionHash(defaultOrder),
+            uint64(defaultOrder.deadline)
+        );
+        return (makerAssetPermit, takerAssetPermit);
+    }
+
+    function _getEIP712Hash(bytes32 structHash) internal view returns (bytes32) {
```

Comment:
Reused the one in `contracts-test/utils/Sig.sol`.

Reply:
No problem! Working on this PR:
[https://github.com/consenlabs/tokenlon-contracts/pull/114](https://github.com/consenlabs/tokenlon-contracts/pull/114)